src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -19,6 +19,7 @@ import { useState } from 'react';
 import ErrorPage from '../ErrorPage';
 import LoadingIndicator from '../../components/LoadingIndicator';
 import { taskUserToAutocompleteOption } from '../../utils/task.utils';
+import { getUserFullName } from '../../../../backend/src/utils/users.utils';
```

Comment:
you can't use this method it's from the backend which runs as a separate process from the frontend